disttask/loaddata/job.go

```diff
@@ -58,16 +61,37 @@ func (ti *DistImporter) Param() *importer.JobImportParam {
 
 // Import implements JobImporter.Import.
 func (ti *DistImporter) Import() {
-	logutil.BgLogger().Info("start distribute load data", zap.Int64("jobID", ti.Job.ID))
+	ti.logger.Info("start distribute load data")
 	ti.Group.Go(func() error {
 		defer close(ti.Done)
 		return ti.doImport(ti.GroupCtx)
 	})
 }
 
 // Result implements JobImporter.Result.
-func (*DistImporter) Result() importer.JobImportResult {
-	return importer.JobImportResult{}
+func (ti *DistImporter) Result() importer.JobImportResult {
+	var result importer.JobImportResult
+	taskMeta, err := ti.getTaskMeta()
+	if err != nil {
+		result.Msg = err.Error()
+		return result
+	}
+
+	ti.logger.Info("finish distribute load data", zap.Any("task meta", taskMeta))
+	var (
+		numWarnings uint64
+		numRecords  uint64
+		numDeletes  uint64
+		numSkipped  uint64
+	)
+	numRecords = taskMeta.Result.ReadRowCnt
+	// todo: we don't have a strict REPLACE or IGNORE mode in physical mode, so we can't get the numDeletes/numSkipped.
```

Comment:
luckily we will use INGEST INTO, no need to keep same result message as LOAD DATA